Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -1760,6 +1810,21 @@ def get_alert_attachments_command(
     )
 
 
+def get_compromised_credentials_command(
+    client: ZFClient,
+    args: dict[str, Any]
+) -> CommandResults:
+    alert_id = args.get("alert_id", "")
+    credentials = client.get_compromised_credentials(alert_id)
+    if credentials:
+        return fileResult(f"breach_{alert_id}.csv", credentials)
+    return CommandResults(
+        readable_output=f"No compromised credentials were found for {alert_id=}",
+        outputs={},
+        outputs_prefix="File"
+    )
```

Comment:
```suggestion
    return CommandResults(
        readable_output=f"No compromised credentials were found for {alert_id=}"
    )
```
when the response is 404 why do we need the output and not just readable_output?

Reply:
didn't know I could omit output